src/actions/entries.js

```diff
@@ -216,10 +217,11 @@ export function loadEntry(collection, slug) {
     const backend = currentBackend(state.config);
     dispatch(entryLoading(collection, slug));
     return backend.getEntry(collection, slug)
-      .then(loadedEntry => (
-        dispatch(entryLoaded(collection, loadedEntry))
-      ))
+      .then(loadedEntry => {
+        return dispatch(entryLoaded(collection, loadedEntry))
+      })
       .catch((error) => {
+        console.error(error);
```

Comment:
Unfortunately it provides no stack trace, and doesn't point to anything that's aware of source maps. But it's better than nothing.